droid-hal-pioneer.spec

```diff
@@ -4,12 +4,9 @@
 %define device pioneer
 %define vendor sony
 
-%define rpm_device h4113
+%define rpm_device pioneer
```

Comment:
This could be removed altogether due to https://github.com/mer-hybris/droid-hal-device/blob/ece4a525f3f27be99b85f1a29bcc0b7e7c7c9d6e/droid-hal-device.inc#L87

Reply:
Ah, true, will fix